WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift

```diff
@@ -258,6 +258,24 @@ extension UIColor {
     static var primaryButtonBorder = accentDark
     static var primaryButtonDownBackground = muriel(color: .accent, .shade80)
     static var primaryButtonDownBorder = muriel(color: .accent, .shade90)
+
+    static var secondaryButtonBackground: UIColor {
+        #if XCODE11
+            if #available(iOS 13, *) {
+                return UIColor(light: .white, dark: .systemGray5)
+            }
+        #endif
+        return UIColor(light: .white, dark: .neutral(.shade80))
```

Comment:
Based on https://github.com/wordpress-mobile/WordPress-iOS/pull/12386#discussion_r317383172 , it probably also apply here?